### PR TITLE
Qingwei/exp immutable selector

### PR DESCRIPTION
**Pull request by qingweibinary**
Do not merge

### CODE REVIEW

File: src/trade/createSelectorForImmutable.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { createSelectorCreator, defaultMemoize } from 'reselect';
+import Immutable from 'immutable';
+
+function createSelectorForImmutable(...args) {
+    return createSelectorCreator(defaultMemoize, (a, b) => a === b || Immutable.is(a, b))(...args);
+}
+export default createSelectorForImmutable;
```
Comment (qingweibinary) [inline src/trade/createSelectorForImmutable.js]: @borisyankov 
This is a selector that compare via deep equality, you can ignore other code

File: src/_store/directSelectors.js [hunk 2]
```diff
@@ -1,64 +1,53 @@
...
-export const workspaceSelector = state => state.workspace;
-export const viewsSelector = state => state.views;
-export const dailyPricesSelector = state => state.dailyPrices;
-export const boughtContractsSelector = state => state.boughtContracts;
-export const portfolioSelector = state => state.portfolio;
-export const feedLicensesSelector = state => state.feedLicenses;
-export const ohlcSelector = state => state.ohlc;
-export const tradingOptionsSelector = state => state.tradingOptions;
-export const statesSelector = state => state.states;
+export function accountSelector(state) { return state.account; }
```
Comment (borisyankov) [inline src/_store/directSelectors.js]: Why this change?

Comment (qingweibinary) [inline src/_store/directSelectors.js]: You can ignore those
